### PR TITLE
Added Spanish - Spain translation

### PR DESCRIPTION
**Pull request by 97carmine**

### CODE REVIEW

**Comment by micwoj92** [comment]
Couple questions and remarks:

1. What is difference between Spanish - Spain (es-ES) and Spanish - Spain (es-ES_tradnl)

2. you should use hex code anyway 
image

3. @rmcpantoja was working on the different Spanish - Spain translation, could you coordinate and discuss with him? Thanks.

**Comment by 97carmine** [comment]
The difference between Spanish - Spain (es-ES) and Spanish - Spain (es-ES_tradnl) is the way to order certain words. It can be said that there are no differences. 

Source: https://spanish.stackexchange.com/a/28071 (Third paragraph)

**Comment by micwoj92** [comment]
I guess it doesn't hurt to have both of them, will review soon.

File: Langs/0C0A.lang [hunk 1]
```diff
@@ -0,0 +1,59 @@
+[MetaData]
+Language=Spanish - Spain
+Translator=97carmine
+Version=1.0
```
Comment (micwoj92) [inline Langs/0C0A.lang:4]: This has to be program version. Otherwise fine.

**Pull request merged**